### PR TITLE
Bold markdown text is now using the correct font

### PR DESCRIPTION
**Pull request by yaustar**
Fixes text that is bold in markdown but don't appear bold on the site

image

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -150,7 +150,7 @@
     color: rgba(255, 255, 255, 0.75);
   }
   header a:hover {
-    color: rgb(255, 255, 255);
+    color: white;
```
Comment (willeastcott) [inline public/css/styles.css:153]: Have you done `npm install` recently? `sass` has been updated recently, which swaps named colors out for the `rgb` form.

Comment (willeastcott) [reply public/css/styles.css:153]: See this: https://github.com/playcanvas/developer.playcanvas.com/commit/dd995d0a9260f74b65b31f3876ac78029d3e160f

**Review by willeastcott — APPROVED** [review]
Approving, but check my comment about CSS color changes before merging. 😄

**Comment by yaustar** [comment]
Fixed!

**Pull request merged**